osgar/drivers/logzeromq.py

```diff
@@ -11,7 +11,7 @@
 
 class LogZeroMQ:
     def __init__(self, config, bus):
-        bus.register('raw')
+        bus.register('raw:null')
```

Comment:
If you want to have this configurable through the config file, just add something to `config` and read it from there here.

Reply:
Ok, that would be option ... specific to this particular module.